src/transactions/feed/TransactionFeed.test.tsx

```diff
@@ -211,4 +251,17 @@ describe('TransactionFeed', () => {
     await waitFor(() => expect(mockFetch).toHaveBeenCalledTimes(2))
     expect(tree.getByTestId('TransactionList').props.data.length).toBe(2)
   })
+
+  it('renders swap feed item correctly', async () => {
+    mockFetch.mockResponse(JSON.stringify(SWAP_MOCK_RESPONSE))
+
+    const tree = renderScreen({})
+
+    await waitFor(() => tree.getByTestId('TransactionList'))
+
+    expect(tree.queryByTestId('NoActivity/loading')).toBeNull()
+    expect(tree.queryByTestId('NoActivity/error')).toBeNull()
+
+    expect(tree).toMatchSnapshot()
```

Comment:
we agreed a while ago (before you got here) to stop using snapshot tests. they're easy to write, but hard to review and easy to break in the future.

i'd suggest testing explicitly what you are looking for (although it feels like you are testing the same thing here compared to the tests in SwapFeedItem and TransactionDetailsScreen) or add the swap transaction to an existing mock response and capture the output there

Reply:
Oh, I see. I removed the snapshot test. I will add an explicit test if I need it!